Code/Framework/AzToolsFramework/AzToolsFramework/ComponentModes/ShapeTranslationOffsetViewportEdit.h

```diff
@@ -21,7 +21,7 @@ namespace AzToolsFramework
         ShapeTranslationOffsetViewportEdit() = default;
 
         // BaseShapeViewportEdit overrides ...
-        void Setup() override;
+        void Setup(const ManipulatorManagerId manipulatorManagerId = g_mainManipulatorManagerId) override;
```

Comment:
suggestion: It might be okay to omit the default parameter here and just expect the callers to pass through the right `manipulatorManagerId` (that is of course unless that makes this change much larger, so potentially it could be removed and the callsites fixed in another PR)

Reply:
Yeah, I was in two minds about whether it would be better to make callers be explicit about what viewport they wanted. It's only in a few places but I might remove it in the next change.

Reply:
Nice one, yeah I think I'd be in favour of that :+1: